tests/data/instances/sentence_selection_instance_test.py

```diff
@@ -95,18 +95,18 @@ def test_pad_adds_empty_sentences(self):
                            'num_sentence_words': 2,
                            'num_sentences': 4})
         assert self.instance.question_indices == [3, 5, 6]
-        assert self.instance.sentences_indices[0] == [3, 4]
-        assert self.instance.sentences_indices[1] == [5, 1]
-        assert self.instance.sentences_indices[2] == [5, 6]
-        assert self.instance.sentences_indices[3] == [0, 0]
+        assert self.instance.sentences_indices[0] == [0, 0]
```

Comment:
I probably would have changed the `truncate` parameter, instead of changing the test, but feel free to leave it if you want.